src/plugins/global-settings/editor-loader.js

```diff
@@ -64,4 +66,11 @@ domReady( () => {
 	document?.body?.appendChild( globalColorWrapper )
 	createRoot( globalTypographyWrapper ).render( <GlobalTypographyStyles /> )
 	createRoot( globalColorWrapper ).render( <GlobalColorStyles /> )
+
+	// Set the default block to stackable/text
+	if ( settings.stackable_text_default_block ) {
+		setTimeout( () => {
+			setDefaultBlockName( 'stackable/text' )
+		} )
+	}
```

Comment:
Instead of adding it here in the global settings, you can create your own "default-text-block" plugin and register it using `registerPlugin`